src/components/Popups/RemoveAccessPopup/RemoveAccessPopup.styles.ts

```diff
@@ -0,0 +1,21 @@
+import { Box } from '@mui/material';
```

Comment:
Could you pls move these to global styled components to reuse them on other components?